src/wgextender/features/flags/FlagRegistration.java

```diff
@@ -14,7 +14,7 @@
 
 public class FlagRegistration {
 
-	@SuppressWarnings("unchecked")
+	//Вот это надо убрать, к чему тут рефлексия? Флаги проще регистрируются
```

Comment:
The reason why reflection exists here is to transform the old instance flags to support reloading of WGExtender

Reply:
OK, I understood

Reply:
Then remove the comment and bring back suppress warnings so it no longer shows as a diff.